packages/material-ui/src/Menu/Menu.d.ts

```diff
@@ -34,6 +34,12 @@ export interface MenuProps
    * Props applied to the [`MenuList`](/api/menu-list/) element.
    */
   MenuListProps?: Partial<MenuListProps>;
+  /**
+   * If `true`, the Menu is a nested menu of a parent menu,
+   * giving it additional events and props in addition to those
+   * of a normal menu.
+   */
+  nestedMenu?: boolean;
```

Comment:
This name needs to be updated.

Reply:
Is this an internal prop? If so it should be undocumented?

Reply:
Could we just check for the existence of the `setParentLastEnteredItemIndex` prop or similar, instead of having an extra boolean?